### PR TITLE
Add Ubuntu 20.04

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Add Ubuntu 20.04

This also starts using our own full tag for that image
cc @RDIL 
Requirements for that
* from tag must be defined in docker file
* short tags must be in the meta.json
* tag formats must be in meta.json
* the script to generate the tags must **not** exist

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

**Comment by RDIL** [comment]
I don't believe I have write access to this branch

**Comment by TravisEz13** [comment]
@RDIL I should merged it soon...
I'm just testing it...

File: release/preview/ubuntu20.04/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Docker image file that describes an Ubuntu18.04 image with PowerShell installed from Microsoft APT Repo
+FROM debian:bullseye-slim AS installer-env
```
Comment (RDIL) [inline release/preview/ubuntu20.04/docker/Dockerfile:2]: Debian?

File: release/preview/ubuntu20.04/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,107 @@
+# Docker image file that describes an Ubuntu18.04 image with PowerShell installed from Microsoft APT Repo
```
Comment (RDIL) [inline release/preview/ubuntu20.04/docker/Dockerfile:1]:
```suggestion
# Docker image file that describes an Ubuntu20.04 image with PowerShell installed from Microsoft APT Repo
```

File: release/preview/ubuntu20.04/docker/Dockerfile (hunk 1 above)
Comment (TravisEz13) [reply release/preview/ubuntu20.04/docker/Dockerfile:2]: I'll change it...

Comment (TravisEz13) [reply release/preview/ubuntu20.04/docker/Dockerfile:2]:
```suggestion
FROM ubuntu:20.04 AS installer-env
```

**Comment by TravisEz13** [comment]
I made the change in a way to ensure it is not breaking.

File: release/lts/debian11/test-deps/meta.json [hunk 3]
```diff
@@ -11,6 +11,6 @@
         "test-deps-debian"
     ],
     "TestProperties": {
-        "size": 316
+        "size": 332
```
Comment (anmenaga) [inline release/lts/debian11/test-deps/meta.json:14]: size of what?

Comment (RDIL) [reply release/lts/debian11/test-deps/meta.json:14]: The finalized image file

Comment (TravisEz13) [reply release/lts/debian11/test-deps/meta.json:14]: It a test most teams like us do to make sure images don't grow unexpectedly

**Pull request merged**